util/storage-samples/storage-owner-sample/src/test/java/org/fido/iot/storage/To2StorageTest.java

```diff
@@ -411,5 +433,391 @@ protected void failed(Exception e) {
+    try {
+
+      server = Server.createTcpServer(args).start();
+      assertDoesNotThrow(
+          ()-> { dbsManager.updateDeviceReplacementGuid(ds,
+              UUID.fromString("f0956089-c0df-4c34-9c61-f460457e87eb"),
+              UUID.fromString("f0956089-c0df-4c34-9c61-f460457e77eb"));
+          }
+      );
+
+    } catch (SQLException throwables) {
+      throwables.printStackTrace();
+    } finally {
+      dbsManager.removeVoucher(ds, UUID.fromString("f0956089-c0df-4c34-9c61-f460457e77eb"));
+      dbsManager.importVoucher(ds, Composite.fromObject(VOUCHER));
+      if (server != null) {
+        server.stop();
+      }
+    }
+
+  }
+
+  @Test
+  void updateMtuTest() {
+
+    String args[] = new String[]{"-tcp", "-tcpAllowOthers", "-ifNotExists", "-tcpPort", DB_PORT};
+    // start the TCP Server
+    Server server = null;
+
+    try {
+
+      server = Server.createTcpServer(args).start();
+      OwnerDbManager dbsManager = new OwnerDbManager();
+      assertDoesNotThrow(
+          ()-> { dbsManager.updateMtu(ds,"OWNER_MTU_THRESHOLD", 5000);
+          }
+      );
+
+    } catch (SQLException throwables) {
+      throwables.printStackTrace();
+    } finally {
+      if (server != null) {
+        server.stop();
+      }
+    }
+
+  }
+
+  @Test
+  void updateWgetVerificationPreferenceTest() {
+    String args[] = new String[]{"-tcp", "-tcpAllowOthers", "-ifNotExists", "-tcpPort", DB_PORT};
+    // start the TCP Server
+    Server server = null;
+
+    try {
+
+      server = Server.createTcpServer(args).start();
+      OwnerDbManager dbsManager = new OwnerDbManager();
+      assertDoesNotThrow(
+          ()-> { dbsManager.updateWgetVerificationPreference(ds, true);
+          }
+      );
+
+    } catch (SQLException throwables) {
+      throwables.printStackTrace();
+    } finally {
+      if (server != null) {
+        server.stop();
+      }
+    }
+  }
+
+  @Test
+  void removeDeviceSviStringTest() {
+    String args[] = new String[]{"-tcp", "-tcpAllowOthers", "-ifNotExists", "-tcpPort", DB_PORT};
+    // start the TCP Server
+    Server server = null;
+
+    try {
+
+      server = Server.createTcpServer(args).start();
+      OwnerDbManager dbsManager = new OwnerDbManager();
+      assertDoesNotThrow(
+          ()-> { dbsManager.removeDeviceSviString(ds,"default_");
+          }
+      );
+
+    } catch (SQLException throwables) {
+      throwables.printStackTrace();
+    } finally {
+      if (server != null) {
+        server.stop();
+      }
+    }
+  }
+
+  @Test
+  void addDeviceTypeCriteriaTest() {
+    String args[] = new String[]{"-tcp", "-tcpAllowOthers", "-ifNotExists", "-tcpPort", DB_PORT};
+    // start the TCP Server
+    Server server = null;
+
+    try {
+
+      server = Server.createTcpServer(args).start();
+      OwnerDbManager dbsManager = new OwnerDbManager();
+      assertDoesNotThrow(
+          ()-> { dbsManager.addDeviceTypeCriteria(ds, "default",
```

Comment:
Is this a positive scenario where you are trying to add a valid device type criteria?

Reply:
Yes.